zuessite/templates/local_settings_template.py

```diff
@@ -1,23 +1,20 @@
 # Local settings for Zues.
+# Automatically generated
 
-# Step 1: generate a secret key and set it here.
-# For example, run 'pwgen -s 50 1' and set the result here.
-# Do not reuse a secret key from another place.
-SECRET_KEY = ''
+SECRET_KEY = '{{ secret_key }}'
 
-# Debug should be False in production
-DEBUG = False
+# Debug should be False in production.
+DEBUG = True
```

Comment:
Default should still be false as it is this branch that might be used in production